ores/wsgi/util.py

```diff
@@ -61,3 +63,18 @@ def format_output(context, scores, model_info, warning=None, notice=None):
         for rev_id in scores[model]:
             output['scores'][context][model]['scores'][rev_id] = scores[model][rev_id]
     return jsonify(output)
+
+
+# Parse values for features / datasources of interest.
```

Comment:
I'll fix all of PEP8 issues in another PR, don't worry. It'll pass without this but I'm saying you're providing documentation, do it in the most proper way.